test/helper.rb

```diff
@@ -19,7 +19,7 @@
 
 PORT        = 6381
 DB          = 15
-TIMEOUT     = Float(ENV['TIMEOUT'] || 0.1)
+TIMEOUT     = Float(ENV['TIMEOUT'] || 1.0)
```

Comment:
The blocking command test cases are occasionally failure when `TIMEOUT` is not specified.